msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -108,4 +117,26 @@ public IAccount getAccount() {
     public String[] getScope() {
         return mAccessToken.getTarget().split("\\s");
     }
+
+    @Nullable
+    @Override
+    public UUID getCorrelationId() {
+        return mCorrelationId;
+    }
+
+    private UUID sanitizeCorrelationId(@Nullable final String correlationId) {
```

Comment:
Could declare return value `@Nullable`

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1063/commits/353f46523435797d9079cbf600a795d288eda4ed